### DISCUSSION TITLE
Does not work with ID `0`

### DISCUSSION THREAD

**Discussion by D3SOX** [clauderic/dnd-kit#856] [Q&A]
Can't move an item with ID `0`
https://codesandbox.io/s/dnd-kit-sortable-id-0-coh1o7?file=/src/App.tsx

Is this intended? I couldn't find something about it in the docs

**Comment by pstrassmann** [comment]
I feel like this very likely has something to do with the existence of an active element being checked by something like:
 `if (active.id) {...}` 
instead of 
`if (active.id !== undefined)`

**Reply by D3SOX** [reply]
Yeah, I think you're right, it happens to all of us 😄
